src/pages/[platform]/build-a-backend/auth/connect-your-frontend/manage-user-attributes/index.mdx

```diff
@@ -101,6 +101,33 @@ Future<void> _signUp({
 }
 ```
 </InlineFilter>
+
+<InlineFilter filters={['swift']}>
+
+```swift
+func signUp(username: String, password: String, email: String, phonenumber: String) async {
```

Comment:
`phonenumber` is not used in the code snippet.
Also it's not following camelCase :P